src/SFA.DAS.EmployerAccounts.Web/Controllers/TransfersController.cs

```diff
@@ -22,12 +23,14 @@ public class TransfersController : Controller
         private readonly ILog _logger;
         private readonly IMapper _mapper;
         private readonly IMediator _mediator;
+        private readonly EmployerFinanceConfiguration _configuration;
 
-        public TransfersController(ILog logger, IMapper mapper, IMediator mediator)
+        public TransfersController(ILog logger, IMapper mapper, IMediator mediator,EmployerFinanceConfiguration configuration)
         {
             _logger = logger;
             _mapper = mapper;
             _mediator = mediator;
+            _configuration = configuration;
```

Comment:
Rather than injecting the configuration here, it is already injected into the mediator handler. Could we add it to the response object?

Reply:
Both percentage are now created in the queryHandler and passed through via the response